### ISSUE TITLE
[1.14.4] Sleeping does nothing, no voting.

### ISSUE THREAD

**Issue by ProsperCraft** [Quetzi/Morpheus#61]
https://gyazo.com/29f699b585171155799ee33a275a0d23

Morpheus-1.14.3-4.1.20

forge-1.14.3-27.0.25

Modpack - https://www.curseforge.com/minecraft/modpacks/the-adventures-of-halk

**Comment by AliveDevil** [comment]
Same here.
Log:
```
Jul 13 18:42:25 minecraft java[1025]: [18:42:25.670] [Server thread/INFO] [morpheus/]: AliveDevil null 1/2 - 0 miners (50%)
Jul 13 18:42:26 minecraft java[1025]: [18:42:26.170] [Server thread/INFO] [morpheus/]: AliveDevil null 1/2 - 0 miners (50%)
```
Config is the default generated one with 50%.
Both players need to be in bed to trigger sleep.

**Comment by olee** [comment]
Same here - definitely broken for 1.14.3

**Comment by Quetzi** [comment]
I've just uploaded a new version (4.1.22) which will hopefully solve this. I'll close this, but if it hasn't fixed it feel free to reopen the issue and I'll look at it again.

**Comment by ProsperCraft** [comment]
Morpheus-1.14.3-4.1.22 does not fix the problem, please reopen.

**Comment by ProsperCraft** [comment]
Still does not work wtih Morpheus-1.14.4-4.1.23

![2019-07-28_15 55 23](https://user-images.githubusercontent.com/4427684/62013463-2c491d80-b150-11e9-94c0-0c622cd9b3ea.png)

**Comment by ProsperCraft** [comment]
Still broken with Morpheus-1.14.4-4.2.24.jar

**Comment by ProsperCraft** [comment]
The percentage command doesn't accept a number after it anymore.

**Comment by tylerwight** [comment]
Can confirm I am seeing this issue also.

Minecraft 1.14.4
Forge 28.0.45

```
> /morpheus percent 20
[16:15:16.875] [Server thread/INFO] [minecraft/DedicatedServer]: Incorrect argument for command
[16:15:16.889] [Server thread/INFO] [minecraft/DedicatedServer]: ...s percent 20<--[HERE]
```

can't change percent with command and whenever anyone sleeps it shows this and does nothing:

```
 [morpheus/]: <username> null 1/2 - 0 miners (50%)

```

**Comment by Revvilo** [comment]
The percent command is written `/morpheus 20` for 20%, so technically it works but just isn't programmed into the autocomplete properly. It still does nothing towards it actually working, though.

**Comment by Quetzi** [comment]
Just uploaded a new version that may fix some of these issues.

**Comment by ProsperCraft** [comment]
Try again  the percentage command is missing, and sleeping does this -

![unknown (1)](https://user-images.githubusercontent.com/4427684/63282352-dc033e00-c26b-11e9-8fd7-330995bc708f.png)

**Comment by Quetzi** [comment]
Whoops, hopefully 4.2.28 fixes more than it breaks :)

**Comment by Yamza** [comment]
So seems like it partially worked, once. One person slept, it gave the vote percent, then kicked him out of bead without changing time. I got in bed after and the time changed to morning. Hasn't worked again since.

**Comment by Kaerius** [comment]
As far as I understand, the mod does not create a configuration file. Can you tell me what config should be? Since the morpheus.toml and morpheus.cfg files do not respond to the morpheus.toml files. 
Forge 28.0.55
Morpheus 4.2.28
![image](https://user-images.githubusercontent.com/1667943/63642133-35d48f80-c6d4-11e9-8159-cba078210e7e.png)
![image](https://user-images.githubusercontent.com/1667943/63642141-5270c780-c6d4-11e9-9b2f-84c425eec162.png)

**Comment by Quetzi** [comment]
The config file now lives inside the world folder, I'm looking into command issues, quite a lot has changed in Forge with commands and it isn't obvious how it works.

**Comment by Quetzi** [comment]
May have nailed it 4.2.33 has more command changes. This is taking a while to figure out because the machine I am doing dev on can't actually run the code in dev, so I'm hampered a bit.

**Comment by Quetzi** [comment]
This may now be solved with the latest build (4.2.38). Confident I've nailed this now so going to close the issue once more, will reopen if it's still an issue.
